carthage.sh

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env bash
+# carthage.sh
+# Usage example: ./carthage.sh build --platform iOS
```

Comment:
Please also update https://github.com/GetStream/stream-chat-swift/blob/main/Sample_v3/README.md with the new script name

Reply:
Updated. We don't actually use carthage for v3 anymore

Reply:
Good, we can probably remove the whole section from the readme then.

Reply:
I did, please check

Reply:
Good. Now after you actually pushed the changes, it's easier for me to review them :trollface: